### PR TITLE
Provide some guidance on getting traffic to/from the calico cluster

### PR DESCRIPTION
**Pull request by tomdee**
@fasaxc Do these iptables commands look OK to you? Do you think they will interact with felix in bad ways?
@lxpollitt Can you review the text?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,3 +87,56 @@ Options:
...
+range (or multiple IP ranges) if you've configured different pools. 
+
+The command will need to be run each time the host is restarted.
+
+### Ingress traffic to containers
+if you're trying to host a public facing service on your Calico network then
+you'll want some way of exposing that service. It's highly desirable that 
+you assign a public IP to your container and peer with your border router 
+so that traffic for that IP is brought to the appropriate compute host. If 
+you can't do this and you're host already has a public facing IP, then you
```
Comment (fasaxc) [inline README.md]: s/you're/your/

File: README.md [hunk 2]
```diff
@@ -87,3 +87,56 @@ Options:
...
+
+### Ingress traffic to containers
+if you're trying to host a public facing service on your Calico network then
+you'll want some way of exposing that service. It's highly desirable that 
+you assign a public IP to your container and peer with your border router 
+so that traffic for that IP is brought to the appropriate compute host. If 
+you can't do this and you're host already has a public facing IP, then you 
+can use port mapping to get the traffic into the container.
+
+You will need to runt he following commands to on the host that the container
```
Comment (fasaxc) [inline README.md]: typo: runt

File: README.md [hunk 3]
```diff
@@ -87,3 +87,56 @@ Options:
...
+is running on.
+
+```
+iptables -A FORWARD -p tcp -d <CALICO_IP> --dport <SERVICE_PORT> -j ACCEPT
```
Comment (fasaxc) [inline README.md]: Why is this FORWARD rule needed; I think it'll just bypass all our security?  If you get the PREROUTING NAT rule right then it should get routed to the VM and accepted/rejected based on its profile rules.

**Comment by fasaxc** [comment]
@tomdee instead of writing a doc about it, why not add the NAT feature to calicoctl?  Feels like only moderately more work, with a lot less room for user mistakes.
```
calicoctl expose port ...
```
or some such.

**Comment by tomdee** [comment]
Adding to calicoctl would be sensible, I'll consider that for a future increment.

File: README.md (hunk 3 above)
Comment (tomdee) [inline README.md]: Getting rid of this FORWARD would be very desirable. Can you see anything obviously wrong with the PREROUTING rule or do you think it has a chance of working without the FORWARD rule in place?

**Pull request merged**